Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -790,15 +790,15 @@ void ProgramShaderCache::CreateHeader()
           "#define UBO_BINDING(packing, x) layout(packing, binding = x)\n"
           "#define SAMPLER_BINDING(x) layout(binding = x)\n"
           "#define TEXEL_BUFFER_BINDING(x) layout(binding = x)\n"
-          "#define SSBO_BINDING(x) layout(binding = x)\n"
+          "#define SSBO_BINDING(x) layout(std430, binding = x)\n"
```

Comment:
This part needs to be checked against `bSupportsFragmentStoresAndAtomics ` probably similar to below